pom.xml

```diff
@@ -539,6 +539,12 @@ Import-Package: \\
           </configuration>
         </plugin>
 
+        <plugin>
+          <groupId>org.openhab.core.tools</groupId>
+          <artifactId>i18n-maven-plugin</artifactId>
+          <version>${project.version}</version>
```

Comment:
@wborn This seems to break our release build as it is a SNAPSHOT dependency, which is not allowed - see https://ci.openhab.org/view/Sandbox/job/sandbox-openhab3-release/439/console.
We probably would have to maintain it in a separate repo just like SAT or only add this dependency after releasing version 3.2.

Reply:
The separate repo is probably the cleanest solution. You could also try to move the whole plugin configuration to a profile that is enabled by default but disabled during release. I'm not 100% sure if the no-snapshot check also includes disabled profiles, though.

Reply:
I've reverted the PR for now. I'm not a big fan of many repos because of all the overhead and when code changes it can only be used after creating a release. Especially for new code like this tweaks may be necessary. :wink:

Reply:
I understand your pov, but afaik you cannot build plugins and use them in the very same reactor. So they will require an independent release cycle - however we achieve this.

Reply:
It could also be an idea to put a different label on the "box" currently called "static-code-analysis" (build-tools?). Then the name would allow for putting more similar stuff into that box. :slightly_smiling_face:

But is still possible that when the data in OH-INF changes, the plugin would start depending on OHC snapshot dependencies to be able to parse it. So you'd not be able to use an updated version of the plugin until OHC is released.